code/game/objects/items/cards_ids.dm

```diff
@@ -757,3 +757,30 @@ update_label("John Doe", "Clowny")
 
 /obj/item/card/id/job/lawyer
 	icon_state = "lawyer"
+
+/obj/item/card/id/pass
+	name = "promotion pass"
+	desc = "A card that, when swiped on your ID card, will grant you all the access. Should not substitute your actual ID card."
+	icon_state = "data_1"
+	registered_name = "Unregistered ID"
+	assignment = "Access Pass"
+
+/obj/item/card/id/pass/afterattack(atom/target, mob/user, proximity)
+	. = ..()
+	if(istype(target,/obj/item/card/id) && proximity)
+		var/obj/item/card/id/idcard = target
+		for(var/ac in access)
+			idcard.access |= ac
+		if(assignment!=initial(assignment))
+			idcard.assignment = assignment
+		if(name!=initial(name))
+			idcard.name = name
+		to_chat(user, "You upgrade your [idcard] with the [name].")
+		log_id("[key_name(user)] added access to '[idcard]' using [src] at [AREACOORD(user)].")
+		qdel(src)
+
+/obj/item/card/id/pass/minisec
+	name = "deputy promotion card"
+	desc = "Swipe it on your ID card to become a Deputy. How does it feel to be at the other side of the stunbaton?"
+	assignment = "Deputy"
+	access = list (ACCESS_SEC_DOORS,ACCESS_MAINT_TUNNELS,ACCESS_COURT,ACCESS_BRIG,ACCESS_WEAPONS)
```

Comment:
This can be used as an actual ID though? This isn't a code 'refactor' or 'tweak'?

Previously access passes could not, by themselves, open doors or act like IDs. Now, the access pass by itself _can_ be used as an ID...

Reply:
```suggestion
	access = list(ACCESS_SEC_DOORS,ACCESS_MAINT_TUNNELS,ACCESS_COURT,ACCESS_BRIG,ACCESS_WEAPONS)
```
I don't see their ability to be used as independent ID cards as a problem. It *is* a refactor in that these items have been moved under a single typepath.